poc_mobile_verifier/src/verifier.rs

```diff
@@ -46,8 +46,11 @@ impl VerifierDaemon {
                 || epoch_since_last_reward_duration >= reward_period
             {
                 let epoch_duration = epoch_since_last_verify_duration.min(verification_period);
+                let last_rewarded_end_time = Utc.timestamp(*self.last_verified_end_time.value(), 0);
```

Comment:
Why is last_rewarded_end_time the same as last_verified_end_time on the line below?

Reply:
Fixed. Should be rewarded time

Reply:
It is now. (There were two)